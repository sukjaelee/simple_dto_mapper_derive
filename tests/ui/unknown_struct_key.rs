use simple_dto_mapper_derive::DtoFrom;

mod types {
    pub struct Source {
        pub id: String,
    }
}

#[derive(DtoFrom)]
#[dto(from = types::Source, extra = true)]
struct Dto {
    id: String,
}

fn main() {}
