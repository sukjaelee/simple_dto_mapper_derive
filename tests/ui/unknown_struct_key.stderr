error: unknown struct-level #[dto(...)] key; expected `from`
  --> tests/ui/unknown_struct_key.rs:10:29
   |
10 | #[dto(from = types::Source, extra = true)]
   |                             ^^^^^
